CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ The format is based on [Keep a Changelog][kac], and this project adheres to
 ## [Unreleased]
 
 ### Fixed
-- Iterating over an empty view now correctly returns no enities.
+- Iterating over an empty view now correctly returns no entities.
```

Comment:
Should be phrased as it just wont iterate and return any results back rather than returns no entities.